contrib/babelfishpg_tsql/src/pl_exec.c

```diff
@@ -6416,6 +6416,20 @@ exec_assign_value(PLtsql_execstate *estate,
 							 errmsg("null value cannot be assigned to variable \"%s\" declared NOT NULL",
 									var->refname)));
 
+				/* Special handling when target variable is babelfish GUC */
+				if(pg_strncasecmp("babelfishpg_tsql", var->refname, 16) == 0)
```

Comment:
What if customer is writing a sql like set babelfishpg_tsql.row_count = 1?
will it affect the rowcount behavior ?

Reply:
That will throw syntax error because grammar is `SET ROWCOUNT (LOCAL_ID | MINUS? DECIMAL)`.

Reply:
what would be varname in case of `set @babelfishpg_tsql_var = 10`??

Reply:
varname will be `@babelfishpg_tsql_var` so there will not any problem.